### PR TITLE
add callTracer

### LINKED ISSUE #2009: support callTracer for debug trace endpoints

### PR DESCRIPTION
**Pull request by tclemos**
Closes #2009.

### What does this PR do?

Add support to callTracer

### Reviewers

Main reviewers:

@ToniRamirezM 
@agnusmor

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 run:
   timeout: 5m
+  skip-dirs:
```
Comment (ToniRamirezM) [inline .golangci.yml:4]: Why are you skipping these folders? I would lint at least our files.

File: state/state.go [hunk 2]
```diff
@@ -36,10 +36,12 @@ import (
...
 const (
 	// Size of the memory in bytes reserved by the zkEVM
-	zkEVMReservedMemorySize int  = 128
-	two                     uint = 2
-	cTrue                        = 1
-	cFalse                       = 0
+	two uint = 2
+)
+
+const (
```
Comment (ToniRamirezM) [inline state/state.go:42]: Can we merge these const sections?

**Review by ToniRamirezM — APPROVED** [review]

File: .golangci.yml (hunk 1 above)
Comment (tclemos) [reply .golangci.yml:4]: This was the quickest solution I've found to avoid Linter complaining about the geth code.

Comment (tclemos) [reply .golangci.yml:4]: let's review this when we decide to merge the release branch into the develop branch.

**Pull request merged**